docs/competition/safeguarding_2020/safeguarding_2020.md

```diff
@@ -0,0 +1,41 @@
+# Safeguarding 2020
+
+Unforeseeable circumstances in SR2020 has created a situation where we need to give further thought to safeguarding as our normal processes no longer  cover all eventualities.
+
+This guidance is purely for SR2020, and must be reviewed for applicability and appropriateness before any future events.
+
+This guidance is largely driven by the fact that we can no longer guarantee the involvement of the competitor's responsible adult in any interactions.
+
+## YouTube comments
+
+Most of the communication during the SR2020 competition is likely to be through comments on the livestream YouTube video.
+
+All normal advice regarding suitability of comments should be observed, in addition the following should be followed:
+
+- Personal contact details/social media accounts should not be shared or used.
+- No private discussions should be held between volunteers and competitors.
+- In the unlikely event that a competitor has to be contacted privately, another adult must be included in the discussion, ideally this should be both the competitor's responsible adult and another SR volunteer.
+- In the event that a competitor makes contact privately, these additional parties must be immediately included.
+
+## Competitor Interviews
+
+Some pre-recorded interviews with competitors are going to be displayed during the events.
+
+During recording of these interviews the following guidance should be observed:
+
+- Calls must involve 2 SR volunteers.
+- Calls must involve the competitor's responsible adult. This could be either their teacher joining the call, or their parent/carer.
+- Both SR volunteers must be in the call before the competitor joins. It is acceptable for this to be achieved merely by both volunteers joining the call at least 5 minutes before the competitor is expected to join.
+- Both SR volunteers and the responsible adult must leave the call after the competitor, excepting the circumstance whereby the call is ended for all parties simultaneously by the host.
+- It is acceptable for the responsible adult to be a parent merely in the room and not constantly visible to the camera. However, this would require the competitor to be on loud speaker (not using headphones), and the parent stating to the camera at the beginning that they will remain in the room throughout.
+- The intended responsible adult should be known in advance, however it is acceptable for it to change last minute.
+- Should the second SR volunteer or the responsible adult leave the call unexpectedly, e.g. through network dropout etc., then the call can continue uninterupted so long as the expectation is for the missing party to rejoin imminently. Should a few minutes pass and the missing party has not rejoined, or should it become apparent that they don't intend to rejoin, then the call should be ended and rescheduled.
+- The entire call should be recorded, and an unmodified original stored appropriately on SR systems.
```

Comment:
There was an extra point you mentioned last week stating that this footage must only be stored on a volunteer's machine if they are editing it for broadcast, and that it must be deleted at the earliest opportunity following that (I think you phrased it better than this)

Reply:
Yes. Though there is a question of where the line is between safeguarding and general data protection in this regard. Arguably it falls under both.
Expect something relating to that in a future PR. It's slightly less timing sensitive.